x-pack/docs/en/security/authorization/document-level-security.asciidoc

```diff
@@ -26,19 +37,37 @@ POST /_security/role/click_role
     }
   ]
 }
---------------------------------------------------
+----
 
-NOTE: Omitting the `query` entry entirely disables document level security for
-      the respective indices permission entry.
+You can write this same query using nested JSON syntax:
 
-The specified `query` expects the same format as if it was defined in the
-search request and supports the full {es} <<query-dsl,query DSL>>.
+[source,console]
+----
+POST _security/role/click_role
+{
+  "indices": [
+    {
+      "names": [
+        "events-*"
+      ],
+      "privileges": [
+        "read"
+      ],
```

Comment:
Nit: Sorry I know this snippet is copied from my previous comment, but I think it would look better and easier to spot the difference if we keep this part consistent to the previous example, i.e.:
```javascript
      ...
      "names": [ "events-* ],
      "privileges": [ "read" ],
      ...
```
The content of the `query` field looks fine as it signifies the nested structure.